device_tags_python/Dockerfile.template

```diff
@@ -1,6 +1,4 @@
-# base-image for python on any machine using a template variable,
-# see more about dockerfile templates here: https://www.balena.io/docs/learn/develop/dockerfile/
-FROM balenalib/%%BALENA_ARCH%%-debian-python:3-bookworm
+FROM python:3-bookworm
 
 # use `install_packages` if you need to install dependencies,
```

Comment:
need to replace these commands with apt-get

Reply:
Also line 19-20 won't work in the official library

Reply:
Good call